### PR TITLE
[5.2] Allow jobs to select the queue connection

### PR DESCRIPTION
**Pull request by GrahamCampbell**
Back again, lol.

### CODE REVIEW

File: src/Illuminate/Bus/Dispatcher.php [hunk 1]
```diff
@@ -104,7 +104,8 @@ protected function commandShouldBeQueued($command)
      */
     public function dispatchToQueue($command)
     {
-        $queue = call_user_func($this->queueResolver);
+        $connection = isset($command->connection) ? $command->connection : null;
```
Comment (taylorotwell) [inline src/Illuminate/Bus/Dispatcher.php]: Space after this line.

Comment (GrahamCampbell) [inline src/Illuminate/Bus/Dispatcher.php]: Thought you might say that, lol.

Comment (GrahamCampbell) [inline src/Illuminate/Bus/Dispatcher.php]: Done. :)

**Comment by taylorotwell** [comment]
Do you want a `onConnection` method on Illuminate\Bus\Queueable?

**Comment by GrahamCampbell** [comment]
I guess we do, yes.

**Comment by GrahamCampbell** [comment]
Done.

**Pull request merged**